go.mod

```diff
@@ -23,5 +23,5 @@ require (
 )
 
 // This version uses broken NeoFS API with incompatible signature
-// definitions. See fix in https://github.com/nspcc-dev/neofs-api/pull/203
+// definitions. See fix in https://github.com/nspcc-dev/frostfs-api/pull/203
 retract v2.12.0
```

Comment:
We are dropping old tags, so we can drop this retract as well.

Reply:
Fixed